pkg/secrets-store/nodeserver_test.go

```diff
@@ -347,20 +368,29 @@ func TestNodePublishVolume(t *testing.T) {
 			}
 
 			for attempts := 2; attempts > 0; attempts-- {
-				// How many times 'total_node_publish' and 'total_node_publish_error' counters have been incremented so far.
+				// How many times 'node_publish_total' and 'node_publish_total_error' counters have been incremented so far.
```

Comment:
Same as above about the name chance

Reply:
This is one of the renamed metric that I mentioned in the other comment.

Here is the existing test checking for the correct metric
https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/608fae1f83980e610251214e69dca264b9e90ec0/test/bats/e2e-provider.bats#L428-L429